sqs/src/test/scala/docs/scaladsl/SqsPublishSpec.scala

```diff
@@ -191,6 +198,30 @@ class SqsPublishSpec extends AnyFlatSpec with Matchers with DefaultTestContext w
     }
   }
 
+  it should "publish batch of SendMessageRequests with individual delays and pull them" taggedAs Integration in {
+    new IntegrationFixture {
+      // #batch-send-request
```

Comment:
these was paradox documentation syntax, could you remove it? because it is duplicated.
```suggestion
```